.gitignore

```diff
@@ -16,6 +16,7 @@ mysql/
 
 # dependencies
 /node_modules
+/package-lock.json
```

Comment:
Yes, it should. CI (dependabot, etc) needs it. The official node documentation say to include it in every repo: https://nodejs.dev/learn/the-package-lock-json-file

Reply:
Apologies yeah, I was reading up on this earlier and reading just that (i.e. that you should include it), dunno why I still included the commit